### ISSUE TITLE
lombok plugin not working, @data getter setter constructor annotations didn't generate code, and compilation error occurs

### ISSUE THREAD

**Issue by zhast460** [mplushnikov/lombok-intellij-plugin#873]
### Short description
lombok plugin seems not working totally. it couldn't recognize Data, Getter, Setter or constructor annotation. No setter getter are generated and source codes are full of red lines of "couldn't resolve method" issues. Compilation errors occur when building project in IDE.

### Version information

 * **IDEA Version:** 
 IntelliJ IDEA 2020.1.4 (Ultimate Edition)
Build #IU-201.8743.12, built on July 21, 2020

 * **JDK Version:** zulu 1.8.0_202
 * **OS Type & Version:** Windows
 * **Lombok Plugin Version:** 0.30-2020.1  (even the latest 2020.2-EAP not work)
 * **Lombok Dependency Version:** 1.18.8

### Details

getter mothod couldn't be resolved:
![image](https://user-images.githubusercontent.com/12053298/89749973-8e2aba00-dafc-11ea-832f-a37bd9a692d1.png)

@Data is used and well defined for the object:
![image](https://user-images.githubusercontent.com/12053298/89750003-abf81f00-dafc-11ea-8bb8-dfeffc3a7418.png)

lombok jar is showing in the project dependency, and I have checked the local gradle cache folder, can see the jar there
![image](https://user-images.githubusercontent.com/12053298/89750510-f084ba00-dafe-11ea-8a21-faf8c684e634.png)

I have tried unstall and reinstall lombok plugin, reload Gradle project, refresh Gradle dependencies, invalidate cache and restart IDE. Both couldn't resolve the issue, please help!

**Comment by meiblorn** [comment]
Seems related to https://github.com/mplushnikov/lombok-intellij-plugin/issues/829

**Comment by unamanic** [comment]
Please ensure that Annotation Processing is Enabled in `Build, Execution, Deployment` > ` Compiler` > `Annotation Processors`:

![image](https://user-images.githubusercontent.com/144254/90434564-3f989500-e093-11ea-841f-cba2c3c28515.png)

**Comment by zhast460** [comment]
> Please ensure that Annotation Processing is Enabled in `Build, Execution, Deployment` > ` Compiler` > `Annotation Processors`:
> 
> ![image](https://user-images.githubusercontent.com/144254/90434564-3f989500-e093-11ea-841f-cba2c3c28515.png)

Hi unamanic, Annotaion processing is always enabled.

**Comment by RavenX447l** [comment]
Do you have any other issues present? It's possible the annotation process is not running due to other build errors.

**Comment by zhast460** [comment]
> Do you have any other issues present? It's possible the annotation process is not running due to other build errors.

Seems don't have other issues. This one is a major blocker and I can't proceed to other operations in IDE

**Comment by unamanic** [comment]
> Hi unamanic, Annotaion processing is always enabled.

This is not the default, I usually have to turn it on for projects that use lombok.  

Are you able to build successfully directly with gradle?

**Comment by zhast460** [comment]
> > Hi unamanic, Annotaion processing is always enabled.
> 
> This is not the default, I usually have to turn it on for projects that use lombok.
> 
> Are you able to build successfully directly with gradle?
> Can you post your build.gradle?

Yes, if I run Gradle command outside of IDE, it can be successfully built.

**Comment by remal** [comment]
I had the same issue and it was related to #829. Updating IDEA to 2020.2.1RC helped me.
